tests/baselines/reference/docker/vscode.log

```diff
@@ -4,24 +4,47 @@ yarn run vX.X.X
 $ gulp compile --max_old_space_size=4095
 [XX:XX:XX] Node flags detected: --max_old_space_size=4095
 [XX:XX:XX] Using gulpfile /vscode/gulpfile.js
-[XX:XX:XX] Error: /vscode/extensions/github-authentication/src/common/utils.ts(64,3): Argument of type '(result: U) => U' is not assignable to parameter of type '(value: awaited U) => U | PromiseLike<U>'.
-  Types of parameters 'result' and 'value' are incompatible.
-    Type 'awaited U' is not assignable to type 'U'.
-      'U' could be instantiated with an arbitrary type which could be unrelated to 'awaited U'.
+[XX:XX:XX] Error: /vscode/src/vs/workbench/contrib/search/common/searchModel.ts(609,21): Type 'FileMatch | FileMatch[]' must have a '[Symbol.iterator]()' method that returns an iterator.
+[XX:XX:XX] Error: /vscode/src/vs/base/browser/ui/tree/asyncDataTree.ts(215,2): Type '{ collapseByDefault: true; identityProvider: { getId(el: IAsyncDataTreeNode<TInput, T>): { toString(): string; }; } | undefined; dnd: AsyncDataTreeNodeListDragAndDrop<TInput, T> | undefined; ... 31 more ...; horizontalScrolling?: boolean | undefined; } | undefined' is not assignable to type 'IObjectTreeOptions<IAsyncDataTreeNode<TInput, T>, TFilterData> | undefined'.
```

Comment:
This looks to actually be due to https://github.com/microsoft/TypeScript/pull/36539